apiserver/facades/client/controller/controller_test.go

```diff
@@ -229,7 +229,7 @@ func (s *controllerSuite) TestListBlockedModels(c *gc.C) {
 				"BlockChange",
 			},
 		},
-		params.ModelBlockInfo{
```

Comment:
These sorts of changes are usually nicer if they are also associated with pulling up the extra } characters on the same line eg:

 []params.ModelBlockInfo{{
  Block: one
  ...
}, {
   Block: two
}

(At least IMO if you're going to dorp off the extra text, it looks like far too much wasted space, and *that* unfortunately 'go fmt' doesn't prescribe one way or the other)

Reply:
I have that same grievance as well, it should automatically align and drop space. I'll see if there are options.